### DISCUSSION TITLE
Unable to generate configuration: dial tcp: lookup jira.companyname.com on 8.8.8.8:53: no such host

### DISCUSSION THREAD

**Discussion by KenzoKenzo94** [ankitpokhrel/jira-cli#572] [Q&A]
Hi,

I am trying to setup the client with a Jira DataCenter implementation. I tried to use both `JIRA_API_TOKEN="password"` and create a PAT using it as a token and the `JIRA_AUTH_TYPE` set to `bearer`. None of the cases work, and I get the error attached:

![image](https://user-images.githubusercontent.com/50076237/218482222-611cb0f8-382d-4457-a9ec-01b397456ae0.png)

My understanding is that the client is trying to resolve a non-public URL using public DNS. Is there a way to change that? Or the issue is somewhere else?

Thank you.

**Comment by ankitpokhrel** [comment]
Looks like this could be similar to https://github.com/ankitpokhrel/jira-cli/issues/444
